cypress/integration/login.js

```diff
@@ -22,11 +20,11 @@ describe('Sourcebooks login', function() {
         loginPage.loginWithUserAndRole("Jelena Černyšova", "Team Lead")
 
         cy.url().should('include', '/time-logging');
-        pageComponents.getPageTitle().contains('Timesheets')
+        timeLoggingPage.getPageTitle().contains('Timesheets')
         timeLoggingPage.getCalendar().should('be.visible')
         timeLoggingPage.getTileForm().should('be.visible')
-        pageComponents.getLoggedInUser().contains('Jelena Cernyšova');
-        pageComponents.getNavigationTabsList().should('have.length', 2)
+        timeLoggingPage.getLoggedInUser().contains('Jelena Cernyšova');

```

Comment:
Why did you change back to using `timeLoggingPage` ? Using `pageComponents` seemed better

Reply:
it uses pageComponents, as TimeLoggingPage class extends pageComponents  class, therefore the logic of pageComponents is in TimeLoggingPage  class without creating additional variables in a test and the logic can still be used

Reply:
Oh, didn't see that, all good then :)